crates/ironrdp-pdu/src/gcc/conference_create.rs

```diff
@@ -221,7 +221,7 @@ impl PduParsing for ConferenceCreateResponse {
         // ConnectData::connectPDU: length (MUST be ignored by the client according to [MS-RDPBCGR])
         per::legacy::write_length(
             &mut stream,
-            gcc_blocks_buffer_length as u16 + CONFERENCE_RESPONSE_CONNECT_PDU_SIZE,
+            gcc_blocks_buffer_length as u16 + CONFERENCE_RESPONSE_CONNECT_PDU_SIZE + 1,
```

Comment:
Good catch. I think this fix will break some tests. FYI a lot of tests in ironrdp-pdu are very old and I wouldn’t be surprised if some of them are snapshots generated using IronRDP itself (i.e.: not official samples from the documentation or extracted from an actual RDP session)

Reply:
I am not sure why WireShark complains about this field though.. I tried to read their code, but it's not obvious :) anyway the field is marked as being ignored in RDP spec, so I guess we can just add +1 to make WS happy.